ubuntu/debian/control

```diff
@@ -17,7 +17,8 @@ Build-Depends: cmake,
                protobuf-compiler,
                python3,
                python3-dev,
-               python3-distutils,
+# distutils is not present on noble onwards
```

Comment:
I think it's okay to remove python3-distutils completely. Probably no need for this logic.